MetaCocktailsSwiftData/Cocktails/Classics/Bamboo.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  Bamboo.swift
+//  MetaCocktailsSwiftData
+//
+//  Created by James on 10/8/23.
+//
+
+import SwiftUI
+
+var Bamboo = Cocktail(cocktailName: "Bamboo",
+                      imageAsset: nil,
+                      glasswareType: Glassware.coupe,
+                      garnish: [.orangeOrLemon],
+                      ice: nil,
+                      author: "Louis Eppinger",
+                      spec: bambooSpec,
+                      buildOrder: nil,
+                      tags: bambooTags)
+
+
+var bambooSpec = [CocktailIngredient(ingredient: .amontillado, value: 1.5, unit: .fluidOunces), 
+                  CocktailIngredient(ingredient: .dryVermouth, value: 1.5, unit: .fluidOunces),
+                  CocktailIngredient(ingredient: .orangeBitters, value: 2, unit: .dash)]
+
```

Comment:
For a cocktailIngredient, .fluidOunces is set in the initializer as the default unit - so we only need to add a unit for other types (.dash, .berries etc)

Reply:
So for any ingredient in fluidOunces we can write this as just CocktailIngredient(ingredient: .amontillado, value: 1.5)